core/src/test/java/com/microsoft/applicationinsights/internal/persistence/LocalFileLoaderTests.java

```diff
@@ -0,0 +1,303 @@
+package com.microsoft.applicationinsights.internal.persistence;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.io.Resources;
+import com.microsoft.applicationinsights.internal.authentication.AadAuthentication;
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+import static com.microsoft.applicationinsights.internal.persistence.PersistenceHelper.DEFAULT_FOlDER;
+import static org.junit.Assert.*;
+
+public class LocalFileLoaderTests {
```

Comment:
for the tests, what about just verify reading/writing a random file? i don't think LocalFileLoader and LocalFileWriter even care what the contents of the bytes/file are?

Reply:
added a test for random content.